native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj

```diff
@@ -364,47 +550,58 @@
 		};
 		DE6EE8CF2254E0630087FD5E /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 2BE3FCE3B52693330E34BDC6 /* Pods-MeetHourSDKTest.debug.xcconfig */;
+			baseConfigurationReference = A8463F6C9C2180CDA6F7C9A4 /* Pods-MeetHourSDKTest.debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_ENTITLEMENTS = MeetHourSDKTest/MeetHourSDKTest.entitlements;
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 7AAZ3ZK627;
 				INFOPLIST_FILE = MeetHourSDKTest/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdkTest;
+				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdk;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = MeethourWatchOS;
+				PROVISIONING_PROFILE_SPECIFIER = MeetHourTestSDK;
 				TARGETED_DEVICE_FAMILY = "1,2";
 			};
 			name = Debug;
 		};
 		DE6EE8D02254E0630087FD5E /* Release */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 4F407C543F8277B0EDFFD647 /* Pods-MeetHourSDKTest.release.xcconfig */;
+			baseConfigurationReference = 1934C7C56416655F8E0261E5 /* Pods-MeetHourSDKTest.release.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_ENTITLEMENTS = MeetHourSDKTest/MeetHourSDKTest.entitlements;
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 7AAZ3ZK627;
 				INFOPLIST_FILE = MeetHourSDKTest/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdkTest;
+				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdk;
```

Comment:
@chepury-bharathraj  - This bundle ID is correct.

Reply:
"**go.meethour.io.ios.sdk**" we have created it for **iOS**.

Reply:
Ok sure. Let's stick to this. @chepury-bharathraj

Reply:
Done.